### DISCUSSION TITLE
Comparison/DE plot file selection

### DISCUSSION THREAD

**Discussion by skiniry** [skiniry/Trips-Viz#7] [Ideas]
The method of file selection on the comparison and differential expression plot pages is quite cumbersome and not intuitive. Depending on the number of files/groups users have to select each file and then press a button, and they may have to do this >10 times. Making a mistake requires the user to clear the group and start from scratch. I think this could be improved a lot by implementing a drag&drop system instead.

**Comment by solenoozerec** [comment]
Agree, this is a big problem that will only get bigger if nothing is changed. I don't have suggestions on how to solve this except for looking for ideas in other related resources like https://www.ebi.ac.uk/gxa/home

**Reply by JackCurragh** [reply]
Drag and drop is definitely doable. I just implemented a version in my app which is essentially three javascript functions. It can definitely be improved upon. I will make a branch when I can get Main branch working on my clone. 

In terms of file selection in general, if we had more meta information stored regarding each file we could improve the selection process even further. But thats a bigger task entirely.

**Reply by solenoozerec** [reply]
Wouldnt it be possible to extract some meta-information directly from GEO (like cells, conditions, etc) and calculate other meta-information ourselves (coverage, periodicity...)

**Reply by JackCurragh** [reply]
Yes, edirect has a tool efetch that will return meta-info from SRA or BioProject that we could then parse to populate a sqlite table.
